_fellows/renata-phillippi.md

```diff
@@ -3,13 +3,17 @@ id: renata-phillippi
+Renata (she/her) began her career in web and print design, then transitioned to User Experience Design after earning an MFA from Parsons School of Design. Before becoming a PIF, she had the opportunity to work across industry, for companies such as Apple, CBS Interactive, Bank of America, and Electronic Arts. She has worked to improve organizational processes and services, design applications, solve user needs, provide research, and build design teams and operations. Renata has also assisted nonprofits to accomplish their business goals, including winning an award for the best Social Justice App assisting the International Justice Mission to rescue more people out of human trafficking. She has consulted with CEOs looking to build startups to tackle some of today’s wicked problems including climate change, food insecurity, and reskilling the workforce. She has also been an NYC Innovation Fellow working to overhaul and digitize city operations impacting the lives of millions of New Yorkers.
+
+As a PIF, in her first year she worked closely with leaders at Veterans Affairs (VA) Veterans Experience Office (VEO) and the Office of the Chief Technology Officer (OCTO) to develop new systems and processes to align all user research within the VA under one roof, reducing rework and saving the agency money. She also created and grew the first Design Operations Community of Practice for all designers within government. She has grown the community from 10 people to 200 representing over 20 federal agencies, 5 states, and 2 city level government agencies. In her second year she's continuing her work with the Design Operations Community and working closely with the innovation portfolio at Technology Transformation Services (TTS) focusing on product strategy for the US Web Design System.
```

Comment:
"As a PIF in her first year, she..."
"five states, and two city ..."
"In her second year, she's..."
"U.S. Web..."